plugins/splunk-on-call/src/components/Incident/IncidentListItem.tsx

```diff
@@ -241,12 +243,16 @@ export const IncidentListItem = ({
 
       {incident.incidentLink && incident.incidentNumber && (
         <ListItemSecondaryAction>
-          <IncidentAction
-            currentPhase={incident.currentPhase || ''}
-            incidentId={incident.entityId}
-            resolveAction={handleResolveIncident}
-            acknowledgeAction={handleAcknowledgeIncident}
-          />
+          {!readOnly ? (
+            <IncidentAction
+              currentPhase={incident.currentPhase || ''}
+              incidentId={incident.entityId}
+              resolveAction={handleResolveIncident}
+              acknowledgeAction={handleAcknowledgeIncident}
+            />
+          ) : (
+            ''
```

Comment:
should use `null` instead as the standard react "just render nothing" marker

Reply:
Oh and you can also do `{!readOnly && <IncidentAction ... />}`, even shorter (`false` also just leads to no output)

Reply:
@freben Thanks for the tips; I believe I've correctly made these changes.